terraform/modules/application/alarms.tf

```diff
@@ -1,3 +1,22 @@
+locals {
+  app_scales_to_zero               = var.out_of_hours_scale_down.enabled ? var.out_of_hours_scale_down.scale_to.app == 0 : false
+  sidekiq_scales_to_zero           = var.out_of_hours_scale_down.enabled ? var.out_of_hours_scale_down.scale_to.sidekiq == 0 : false
+  app_desired_count_for_alarms     = (!var.out_of_hours_scale_down.enabled || local.app_scales_to_zero) ? var.app_task_desired_count : var.out_of_hours_scale_down.scale_to.app
+  sidekiq_desired_count_for_alarms = (!var.out_of_hours_scale_down.enabled || local.sidekiq_scales_to_zero) ? var.sidekiq_task_desired_count : var.out_of_hours_scale_down.scale_to.sidekiq
+}
+
+check "suppress_missing_alarm_data_when_scaling_to_zero" {
```

Comment:
This doesn't block you from continuing when these fail, but will show a warning.